### PR TITLE
docfix for ssh_args FAQ

### LINKED ISSUE #79264: Docs request: ssh_args in Ansible Configuration Settings
### Summary

In the FAQ there's a section talking about `ssh_args`: https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-get-ansible-to-notice-a-dead-target-in-a-timely-manner

However, when you go to the config reference: https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-configuration-settings

There is no mention of `ssh_args`.  Can we add `ssh_args` to the Ansible Configuration Settings page?

It seems in 2.4 it was documented, not sure why it went away: https://docs.ansible.com/ansible/2.4/intro_configuration.html#ssh-args

### Issue Type

Documentation Report

### Component Name

unsure

### Ansible Version

```console
$ ansible --version

"latest" in docs
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all

n/a
```

### OS / Environment

n/a

### Additional Information

n/a

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by prasadpatil49**
##### SUMMARY

Doc fix for ssh_args FAQ

Fixes #79264 

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs/docsite/rst/reference_appendices/faq.rst

### CODE REVIEW

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 1]
```diff
@@ -138,7 +138,7 @@ or globally by setting ``ssh_args`` in ``ansible.cfg``.
 How do I get Ansible to notice a dead target in a timely manner?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` from ``ansible.cfg``. Without this option,
+You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` parameter from ``[ssh_connection]`` section of ``ansible.cfg``. Without this option,
```
Comment (bcoca) [inline docs/docsite/rst/reference_appendices/faq.rst]: ansible.cfg is just 'one way' to do it, I would just point/link to the ssh plugin setting.

**Review by bcoca — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/reference_appendices/faq.rst (hunk 1 above)
Comment (prasadpatil49) [reply docs/docsite/rst/reference_appendices/faq.rst]: should I use this link ?  [ssh_plugin](https://docs.ansible.com/ansible-core/devel/collections/ansible/builtin/ssh_connection.html#parameter-ssh_args)

Comment (bcoca) [reply docs/docsite/rst/reference_appendices/faq.rst]: yep, that point at all the ways you can configure this, not just the ansible.cfg entry

**Review by bcoca — APPROVED** [review]

File: docs/docsite/rst/reference_appendices/faq.rst [hunk 2]
```diff
@@ -138,7 +138,7 @@ or globally by setting ``ssh_args`` in ``ansible.cfg``.
 How do I get Ansible to notice a dead target in a timely manner?
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` from ``ansible.cfg``. Without this option,
+You can add ``-o ServerAliveInterval=NumberOfSeconds`` in ``ssh_args`` parameter of `SSH connection plugin <https://docs.ansible.com/ansible-core/devel/collections/ansible/builtin/ssh_connection.html#parameter-ssh_args>`_. Without this option,
```
Comment (oraNod) [inline docs/docsite/rst/reference_appendices/faq.rst]:
```suggestion
You can add ``-o ServerAliveInterval=NumberOfSeconds`` with the ``ssh_args`` parameter in `SSH connection plugin <https://docs.ansible.com/ansible-core/devel/collections/ansible/builtin/ssh_connection.html#parameter-ssh_args>`_. Without this option,
```

**Review by oraNod — COMMENTED** [review]
Thanks very much @prasadpatil49 A few minor suggestions for grammar.

**Review by oraNod — APPROVED** [review]

**Comment by oraNod** [comment]
Thanks @prasadpatil49 ! :tada:

**Pull request merged**